src/test/java/lihua/tasks/DeadlineTest.java

```diff
@@ -0,0 +1,4 @@
+package lihua.tasks;
+
+public class DeadlineTest {
```

Comment:
Would like to clarify the purpose of this file since it is essentially an empty file?

Reply:
Oops I added the test class but forgot to add content to it. Thanks for pointing it out!

Reply:
Also agree :)) Just for me to add 1 comment. TY